localstack/utils/common.py

```diff
@@ -1689,17 +1695,18 @@ def all_exist(*files):
     def store_cert_key_files(base_filename):
         key_file_name = "%s.key" % base_filename
         cert_file_name = "%s.crt" % base_filename
-        # TODO: Cleaner code to load the cert dinamically
+        # TODO: Cleaner code to load the cert dynamically
         # extract key and cert from target_file and store into separate files
         content = load_file(target_file)
-        key_start = re.search(r"-----BEGIN(.*)PRIVATE KEY-----", content)
+        print(content)
```

Comment:
Any reason for this print here, or is this a leftover?

Reply:
Thanks! 👍